### ISSUE TITLE
Content.From.Name not serialized

### ISSUE THREAD

**Issue by JohannesRudolph** [SparkPost/csharp-sparkpost#15]
I'm creating a transmission like this:

```
  var mail = new Transmission();
            mail.Content.From.Email = "test@example.com";
            mail.Content.From.Name = "Tester";
            mail.Content.Subject = subject;
            mail.Content.Text = body;

            mail.Recipients.Add( recipient );
```

However, the json submitted to the API by the sdk/library only has the email in it, and ignores the mail.Content.From.Name property. => Bug

**Comment by JohannesRudolph** [comment]
This serves me as a temporary workaround: 
```
        private void WorkaroundFromHeaderFormatting( Transmission mail )
        {
            // see https://github.com/SparkPost/csharp-sparkpost/issues/15
            mail.Content.From.Email = string.Format( "\"{0}\" <{1}>", mail.Content.From.Name, mail.Content.From.Email );
        }
```

**Comment by darrencauthon** [comment]
I believe I have this handled in #2 .  I originally wrote the mapping of the objects to the API by hand, and in this case I think I just mapped the from address instead of the full offering of the API (which conditionally allows either a string email or an object with many details).

I *think* that PR is ready to go, and the tests pass and it looks good to the eye.  I have yet to actually push emails through these changes, though.  Once I do that, I'll deploy to Nuget and let you know here.

**Comment by darrencauthon** [comment]
I believe this is resolved with 1.0.4, released just this morning.  Thank you for pointing it out!

I tested it with this code:

```c#
var transmission = new Transmission();
transmission.Content.From.Name = "Darren Cauthon";
transmission.Content.From.Email = "testing@sparkpostbox.com";

transmission.Content.Subject = "Checking for a recipient name";
transmission.Content.Text = "1.0.3";
transmission.Content.Html = "1.0.3";

var recipient = new Recipient
{
	Address = new Address { Email = "xxxxxxxxxxxxxxxxx" }
};
transmission.Recipients.Add(recipient);

var client = new Client("xxxxxxxxxxxxxxxxxxxx");
var result = client.Transmissions.Send(transmission);
```

With version 1.0.3 (and earlier), the from name would not be sent.  

With version 1.0.4, the name is sent properly.  The headers_to properly will also be sent, I just didn't fire it in my example.

Firing this code against 1.0.3 and 1.0.4:

![fullscreen_032516_084914_am](https://cloud.githubusercontent.com/assets/148768/14045241/746bcec0-f266-11e5-98ee-13265a242a84.jpg)
